Makefile

```diff
@@ -18,8 +18,8 @@
 STAGINGIMAGE=${GCE_PD_CSI_STAGING_IMAGE}
 STAGINGVERSION=${GCE_PD_CSI_STAGING_VERSION}
 
-PRODIMAGE=gcr.io/google-containers/volume-csi/gcp-compute-persistent-disk-csi-driver
-PRODVERSION=v0.2.0.beta
+PRODIMAGE=gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver
```

Comment:
We don't use these settings to create the official images. Should we just remove these lines?

Reply:
I'm worried we may accidentally manually push to it if it's here.